src/common/components/Input/index.tsx

```diff
@@ -187,8 +175,23 @@ export const Input = forwardRef<HTMLInputElement, InputProps>(({ label, inputWid
                         </InputLabel>
                     )}
 
-                    <Field inputWidth={inputWidth} >
-                        {hide && <IconSpan show={show} onClick={setShow(!show)}>{show ? viewOn : viewOff}</IconSpan>}
+                    {togglePassword ? (
+                        <Field togglePasswordWidth={togglePasswordWidth} >
+                            {togglePassword && <IconSpan show={show} onClick={() => setShow(!show)}>{show ? ViewOn : ViewOff}</IconSpan>}
```

Comment:
`togglePassword && ` seems superfluous as we already check it on row 178. It should be enough with:

```suggestion
                            <IconSpan show={show} onClick={() => setShow(!show)}>{show ? ViewOn : ViewOff}</IconSpan>
```